langchain/vectorstores/redis.py

```diff
@@ -273,7 +273,10 @@ def from_texts(
         try:
             # We need to first remove redis_url from kwargs,
             # otherwise passing it to Redis will result in an error.
-            kwargs.pop("redis_url")
+            try:
```

Comment:
instead of try/except could we explicitly check
```python
if "redis_url" in kwargs:
  del kwargs["redis_url"]
```

Reply:
Yes, that makes sense. I've updated to using an if statement. I've kept the original `pop` syntax though rather than moving to `del` as it seemed to work fine as it was.